content/kubermatic/main/architecture/_index.en.md

```diff
@@ -7,11 +7,11 @@ weight = 5
 
 Kubermatic Kubernetes Platform (KKP) makes full use of Kubernetes cluster to organize and scale workloads, depending on your and your customer's needs. In a typical small-scale setup, pictured below, a single cluster contains KKP and the master components for every user cluster.
 
-![KKP Architecture Diagram](/img/kubermatic/main/architecture/combined-master-seed.png)
+![KKP Architecture Diagram](/img/kubermatic/main/architecture/high-level-architecture.png)
```

Comment:
Does it make sense to drop this diagram completely? It's removed as a file later on, but wouldn't it still be useful to showcase the "combined master/seed" setup?

Reply:
Maybe @csengerszabo can chime in here. I remember that it was in the original task to replace the architecture picture, but tbh I am not fully sure what the intention of the new one was

Reply:
Can we then somehow display both? The original is the general architecture, and this one is more likely an example

Reply:
I have pushed a proposal how we can do this now:

@embik For now I have worded it with “the master cluster” and then used plural for seed- and user-clusters. Is this correct, or is there a possibility for a customer to use more than one master and it is still considered a single KKP installation?

@csengerszabo I have noticed that in the high-level diagram the "master cluster" is called "root cluster". Is this on purpose? If not we should change it in the diagram to have consistent naming. I can do this, but I would need write permissions on the Google Doc to change the diagram.

Reply:
Another question on this diagram, it mentions "Backup to A/B/C" for user clusters. This seems to suggest to me that you can back up / fall over to other seeds (because they are also named A/B/C), which is not the case. So I'm not sure what the diagram is saying here?